lib/ansible/modules/messaging/rabbitmq_binding.py

```diff
@@ -253,6 +242,8 @@ def create(self):
                                                   urllib_parse.quote(self.destination, safe=''))
         self.api_result = self.request.post(self.url,
                                             auth=self.authentication,
+                                            verify=self.cacert,
```

Comment:
Bit late to the party, but `self.cacert` is never set, only `self.verify`, so that seems a bug. And as far as I can see verify is not a kwarg that request.post actually uses, it only accepts `client_cert` and `client_key`.

Reply:
Interesting, good catch.
Fix on the way

Reply:
@hyperized `requests.post` does accept `verify`
https://github.com/requests/requests/blob/master/requests/api.py#L111
https://github.com/requests/requests/blob/master/requests/api.py#L41'
https://github.com/requests/requests/blob/master/requests/sessions.py#L490

Reply:
@Jorge-Rodriguez Thank you for verifying, I wasn't sure!

Reply:
Regarding `requests`, isn't this module using: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py#L888 ?

Reply:
Well, now I'm not sure. But I've been using this code for two years now and it does work.
Let me see.